SPECS/kernel/config

```diff
@@ -2735,6 +2736,7 @@ CONFIG_MLX5_CORE_EN_DCB=y
 CONFIG_MLX5_CORE_IPOIB=y
 CONFIG_MLX5_EN_IPSEC=y
 CONFIG_MLX5_SW_STEERING=y
+# CONFIG_MLX5_HW_STEERING is not set
```

Comment:
result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/b2c793e944f9fd609d7eb2c08eb929a8aeace810